playbooks/ansible-security-trendmicro-appliance/pre.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- hosts: controller
+  tasks:
+    - name: Ensure tox
+      include_role:
+        name: ensure-tox
+
+    - name: Setup tox role
+      include_role:
+        name: tox
+      vars:
+        tox_envlist: venv
+        tox_extra_args: -vv -- ansible-playbook -v -c network_cli playbooks/ansible-security-trendmicro-appliance/files/bootstrap.yaml
+        tox_install_siblings: false
+        zuul_work_dir: "{{ zuul.projects['github.com/ansible/ansible-zuul-jobs'].src_dir }}"
+
+    - name: Install PostgreSQL
```

Comment:
it might be better to move this into the collection repo itself, but lets start here to ensure it works